mltensor-explainer.md

```diff
@@ -203,18 +203,18 @@ A privacy-conscious user wants to perform real-time selfie segmentation of a vid
 
 Currently, using WebNN for this task would require - for each frame - an expensive readback of `GPUBuffer` data to script, uploading the data to the ML context device (which may be the same GPU!), copying the result back to script, and then uploading the frame to be rendered back into a `GPUBuffer`.
 
-An `MLTensor` may be imported into WebGPU, minimizing the number of buffer copies required to render the results of some ML compute. Zero-copy buffer sharing between the two APIs may be supported in some cases.
+An `MLTensor` may be exported into WebGPU, minimizing the number of buffer copies required to render the results of some ML compute. Zero-copy buffer sharing between the two APIs may be supported in some cases.
 
 ```js
 // Create a couple MLTensors to be shared with WebGPU.
-const mlTensor1 = await mlContext.createTensor({..., importableToWebGPU: true});
-const mlTensor2 = await mlContext.createTensor({..., importableToWebGPU: true});
+const mlTensor1 = await mlContext.createTensor({..., exportableToWebGPU: true});
```

Comment:
Bikeshedding: Do we want to remove "Web" from the API so that this is `exportableToGPU` and `exportToGPU` (or maybe `exportableToGPUBuffer` and `exportToGPUBuffer` if we want to be more specific about the type). The rationale is that in the WebGPU API none of the methods or interfaces actually contain the string "WebGPU".

Reply:
Great idea, SGTM.